cg/store/status_case_sample_filters.py

```diff
@@ -23,23 +23,27 @@ def get_cases_associated_with_sample_by_entry_id(
 
 
 def apply_case_sample_filter(
-    functions: List[str],
+    functions: List[Callable],
     case_samples: Query,
     case_id: Optional[str] = None,
     sample_entry_id: Optional[int] = None,
     sample_id: Optional[str] = None,
 ) -> Query:
     """Apply filtering functions to the sample queries and return filtered results."""
-    filter_map = {
-        "get_cases_associated_with_sample": get_cases_associated_with_sample,
-        "get_cases_associated_with_sample_by_entry_id": get_cases_associated_with_sample_by_entry_id,
-        "get_samples_associated_with_case": get_samples_associated_with_case,
-    }
+
     for function in functions:
-        case_samples: Query = filter_map[function](
+        case_samples: Query = function(
             case_samples=case_samples,
             case_id=case_id,
             sample_entry_id=sample_entry_id,
             sample_id=sample_id,
         )
     return case_samples
+
+
+class CaseSampleFilters(Enum):
+    get_samples_associated_with_case: Callable = get_samples_associated_with_case
+    get_cases_associated_with_sample: Callable = get_cases_associated_with_sample
+    get_cases_associated_with_sample_by_entry_id: Callable = (
```

Comment:
Same here, the standard implemented in other classes was to name these variables with capital letters